addons/medical_vitals/functions/fnc_handleUnitVitals.sqf

```diff
@@ -53,7 +53,7 @@ if (_hemorrhage != GET_HEMORRHAGE(_unit)) then {
 
 private _bloodLoss = GET_BLOOD_LOSS(_unit);
 if (_bloodLoss > 0) then {
-    _unit setVariable [QGVAR(bloodloss), _bloodLoss, _syncValues];
+    _unit setVariable [QGVAR(bloodloss), _bloodLoss, _syncValues]; // ToDo: nothing uses this, remove?
```

Comment:
Maybe it should be used to enter unconscious state due to blood loss?

Reply:
`#define GET_BLOOD_LOSS(unit)        ([unit] call EFUNC(medical_status,getBloodLoss))`
that function has fairly complex calcs, and is called in 10 different places
so maybe we could swap it to use the setVar

Reply:
I remember this is something I noticed way back too. If I remember right, I determined that that function was probably intended for the vitals loop only and everywhere else was probably meant to be reading the variable.

Reply:
See 7a87aad
We can calculate wound blood loss only on events that would change it